### ISSUE TITLE
ERM - Fix erm_agreement_cancellation_dates.sql

### ISSUE THREAD

**Issue by stdombek** [folio-org/folio-analytics#619]
[Query](https://github.com/folio-org/folio-analytics/blob/main/sql_metadb/report_queries/erm/resource_lists/erm_agreement_cancellation_dates.sql) stops with error: "operator does not exist: text = uuid"

**Comment by stdombek** [comment]
Directory structure still needs to be discussed in wg erm. Issue moved to release 1.5.

**Comment by stdombek** [comment]
PR is closed. The fix will be submitted with a new PR as soon as issue https://github.com/folio-org/folio-analytics/issues/664 (New directory structure for sql_metadb/report_queries/erm/) is merged.

**Comment by stdombek** [comment]
blocked by #666 (ERM - New directory structure)
